sycl/unittests/program_manager/passing_link_and_compile_options.cpp

```diff
@@ -0,0 +1,529 @@
+//==---- passing_link_and_compile_options.cpp --- Test for passing link and
+// compile options for online linker and compiler ---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include <CL/sycl.hpp>
+
+#include <helpers/CommonRedefinitions.hpp>
+#include <helpers/PiImage.hpp>
+#include <helpers/PiMock.hpp>
+
+#include <gtest/gtest.h>
+
+std::string current_link_options, current_compile_options;
+
+class EAMTestKernel1;
```

Comment:
Just curious, what does `EAM` stand for ?

Reply:
That kernel was used from another test.